contracts/starknet/lib/voting.cairo

```diff
@@ -486,8 +479,7 @@ namespace Voting {
 
         // Make sure `choice` is a valid choice
         with_attr error_message("Voting: Invalid choice") {
-            assert_le(Choice.FOR, choice);
-            assert_le(choice, Choice.ABSTAIN);
+            assert (choice - Choice.ABSTAIN) * (choice - Choice.FOR) * (choice - Choice.AGAINST) = 0;
```

Comment:
this is 2.5x more gas efficient than the 2 `assert_le` comparisons.

Reply:
nice :p
I really need to get used to dealing with prime fields :D